frontend/coverage/clover.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
per eugene's instruction, put the `coverage/` folder in `.gitignore`.  we don't want this to get committed to the repo

https://github.com/Incredihire/Survey/blob/main/.gitignore

Reply:
resolved